## 1.The time required for servicing transmissions is normally distributed with μ = 45 minutes and σ = 8 minutes. The service manager plans to have work begin on the transmission of a customer’s car 10 minutes after the car is dropped off and the customer is told that the car will be ready within 1 hour from drop-off. What is the probability that the service manager cannot meet his commitment? 

In [1]:
from scipy import stats
from scipy.stats import norm

In [2]:
# Find Z-Scores at X=50; Z = (X - µ) / σ 
Z=(50-45)/8
Z

0.625

In [3]:
# Find probability P(X>50) = 1-stats.norm.cdf(abs(z_score))
1-stats.norm.cdf(abs(0.625))

0.26598552904870054

In [4]:
# OR WE CAN FIND IT IN THAT WAY WITHOUT CALCULATING Z SCORE
1-stats.norm.cdf(50,loc=45,scale=8)

0.26598552904870054

In [5]:
# Find probability P(X<=50) = stats.norm.cdf(abs(z_score))
stats.norm.cdf(abs(0.625))

0.7340144709512995

In [8]:
# P(X>50) = 1 - P(X<=50)
1-stats.norm.cdf(abs(0.625))

0.26598552904870054

## 2.	The current age (in years) of 400 clerical employees at an insurance claims processing center is normally distributed with mean μ = 38 and Standard deviation σ =6. For each statement below, please specify True/False. If false, briefly explain why.
* A.	More employees at the processing center are older than 44 than between 38 and 44.
* B.	A training program for employees under the age of 30 at the center would be expected to attract about 36 employees.


#### A. More employees at the processing center are older than 44 than between 38 and 44.


In [10]:
# p(X>44) probability of  employees greater than the age of 44
1-stats.norm.cdf(44,loc=38,scale=6)

0.15865525393145707

In [12]:
# p(38<X<44); probability of Employees between 38 to 44 yrs of age
stats.norm.cdf(44,loc=38,scale=6)-(1-stats.norm.cdf(38,loc=38,scale=6))

0.3413447460685429

### So the probability of number of employees between 38-44 years of age = 34.13%
### Therefore the statement More employees at the processing center are older than 44 than between 38 and 44 is TRUE

#### B. A training program for employees under the age of 30 at the center would be expected to attract about 36 employees.


In [13]:
# P(X<30) probability of Employees under the age of 30 
stats.norm.cdf(30,loc=38,scale=6)


0.09121121972586788

In [15]:
# No. of employees  attending training program from 400 nos. is N*P(X<30)
400*stats.norm.cdf(30,loc=38,scale=6)

36.484487890347154

### So the number of employee with probability 0.912 of them being under age 30=36.48%

### Therefore the statement B of the question is also TRUE

## 4.	Let X ~ N(100, 202). Find two values, a and b, symmetric about the mean, such that the probability of the random variable taking a value between them is 0.99. 
A.	90.5, 105.9 
B.	80.2, 119.8 
C.	22, 78 
D.	48.5, 151.5 
E.	90.1, 109.9


In [2]:
from scipy import stats
from scipy.stats import norm

In [3]:
stats.norm.interval(0.99,100,20)

(48.48341392902199, 151.516586070978)

a= 48.5 b=151.5  option D

## 5.	Consider a company that has two different divisions. The annual profits from the two divisions are independent and have distributions Profit1 ~ N(5, 32) and Profit2 ~ N(7, 42) respectively. Both the profits are in S Million. Answer the following questions about the total profit of the company in Rupees. Assume that S1 = Rs. 45
### A.	Specify a Rupee range (centered on the mean) such that it contains 95% probability for the annual profit of the company.
### B.	Specify the 5th percentile of profit (in Rupees) for the company
### C.	Which of the two divisions has a larger probability of making a loss in a given year?


In [4]:
import numpy as np
from scipy import stats
from scipy.stats import norm

In [5]:
# Mean profits from two different divisions of a company = Mean1 + Mean2
Mean = 5+7
print('Mean Profit is Rs', Mean*45,'Million')  # given assumption is 1 million = Rs.45

Mean Profit is Rs 540 Million


In [6]:
# Variance of profits from two different divisions of a company = SD^2 = SD1^2 + SD2^2
SD = np.sqrt((9)+(16))
print('Standard Deviation is Rs', SD*45, 'Million')

Standard Deviation is Rs 225.0 Million


In [7]:
# A. Specify a Rupee range (centered on the mean) such that it contains 95% probability for the annual profit of the company.
print('Range is Rs',(stats.norm.interval(0.95,540,225)),'in Millions')

Range is Rs (99.00810347848784, 980.9918965215122) in Millions


In [8]:
# B. Specify the 5th percentile of profit (in Rupees) for the company
# To compute 5th Percentile, we use the formula X=μ + Zσ; wherein from z table, 5 percentile = -1.645
X= 540+(-1.645)*(225)
print('5th percentile of profit (in Million Rupees) is',np.round(X,))

5th percentile of profit (in Million Rupees) is 170.0


In [9]:
# C. Which of the two divisions has a larger probability of making a loss in a given year?
# Probability of Division 1 making a loss P(X<0)
stats.norm.cdf(0,5,3)

0.0477903522728147

In [10]:
# Probability of Division 2 making a loss P(X<0)
stats.norm.cdf(0,7,4)

0.040059156863817086